exdb/models.py

```diff
@@ -1,3 +1,88 @@
 from django.db import models
+from django.utils.timezone import now
+from django.conf import settings
 
-# Create your models here.
+
+class SubType(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Type(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Audience(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Organization(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Keyword(models.Model):
+    name = models.CharField(max_length=300)
+
+    def __str__(self):
+        return self.name
+
+
+class Experience(models.Model):
+    STATUS_TYPES = (
+        ('dr', 'Draft'),
+        ('pe', 'Pending'),
+        ('ap', 'Approval'),
+        ('de', 'Denied'),
+        ('ad', 'Approved'),
```

Comment:
I thought 'Pending' and 'Approval' were supposed to be one status of 'Pending Approval', then when it's approved it just gets the 'Approved' status?